### ISSUE TITLE
Improved AMI results?

### ISSUE THREAD

**Issue by picheny-nyu** [espnet/espnet#2371]
I ran the basic AMI recipe and seem to be getting WERs of 21.1. The best numbers in the "results" file in the published recipe seem to be 25% at best, or worse. Has the system significantly improved since the AMI recipe was published (and if so, how?)

**Comment by sw005320** [comment]
Yes, I think this would happen because we have some internal changes, e.g., initialization, CTC loss functions, as espnet or pytorch versions are updated. Such changes could affect the performance significantly for small amounts of a corpus such as AMI.

**Comment by b-flo** [comment]
Also, it seems that the best reported system with data augmentation is trained for 100 epochs while current config has epochs set to 200.
